packages/block-library/src/buttons/editor.scss

```diff
@@ -0,0 +1,78 @@
+.wp-block-buttons .block-editor-block-list__block[data-type="core/button"] {
+	display: inline-block;
+}
+
+
+.wp-block-buttons {
+	// 1. Reset margins on immediate innerblocks container.
```

Comment:
Can we extract all of this to `InnerBlocks horizontalDisplay`?

Reply:
I extracted this CSS logic into generic InnerBlocks flags.